frame/contracts/proc-macro/src/lib.rs

```diff
@@ -510,6 +532,34 @@ fn expand_impls(def: &mut EnvDef) -> proc_macro2::TokenStream {
 /// }
 /// ```
 ///
+/// In legacy versions of pallet_contracts, it was a naming convention that all host functions had
+/// to be named with the `seal_` prefix. For the sake of backwards compatibility, each host function
+/// now can get a such prefix-named alias function generated by marking it by the
+/// `#[prefixed_alias]` attribute:
+///
+/// ## Example
+///
+/// ```nocompile
```

Comment:
Let's make it at least compile able. That should be possible with too much effort.

```suggestion
/// ```no_run
```